distributed/nanny.py

```diff
@@ -162,7 +162,8 @@ def __init__(
         self.death_timeout = parse_timedelta(death_timeout)
 
         self.Worker = Worker if worker_class is None else worker_class
-        self.env = env or {}
+        self.env = env or dask.config.get("distributed.nanny.environ")
```

Comment:
```suggestion
        self.env = dask.config.get("distributed.nanny.environ")
        self.env.update(env or {})
```
Also I could not find anything that passes the env parameter to the Nanny constructor. Maybe we could deprecate it? Unless some dask users actually create Nanny objects by hand?

Reply:
I'm inclined to leave the deprecation question to other work if that's ok.  It seems harmless to leave here for now and there are other things that seem higher priority.